ecommerce/processes/test/test_helper.rb

```diff
@@ -1,6 +1,7 @@
 require "minitest/autorun"
 require "mutant/minitest/coverage"
 require "infra"
+require "active_support/all"
```

Comment:
Why do we need this?

Reply:
It was added because of this failure - https://github.com/RailsEventStore/ecommerce/actions/runs/3906801621/jobs/6675357635.

But after I've moved the solution to `rails_application` completely it should be not needed. I'll remove it.